assets/src/edit-story/components/canvas/layout.js

```diff
@@ -98,6 +98,21 @@ const PageAreaFullbleedContainer = styled(Area).attrs({
   display: flex;
   justify-content: center;
   align-items: center;
+
+  ${({ isBackgroundSelected, theme }) =>
+    isBackgroundSelected &&
+    `
+    &:before {
+      content: '';
+      position: absolute;
+      top: -4px;
+      left: -4px;
+      right: -4px;
+      bottom: -4px;
+      border: ${theme.colors.border.selection} 1px solid;
+      border-radius: 10px;
```

Comment:
We need a fix for the frame as well, by the way - this looks silly:

![page-frame](https://user-images.githubusercontent.com/637548/107530373-cc776380-6b89-11eb-91b4-72615c16dae1.gif)

Reply:
Okay, ignore this one for now - will create a new ticket for hover frames ([there's a new design for it](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=5646%3A2))

Reply:
The hovering also needs to include drop-targeting design at the same time since this blue line is actually drop-target indicator and not related to hovering any element! e.g. text element hover looks a bit different and is handled elsewhere as far as I remember. When I asked Sam, she mentioned that drop-targeting will stay the same but we should still confirm that part (the target indicator part).

Reply:
Yes, I'm aware, thanks. I've made a note of this missing ticket in my list and will create them tomorrow.